scripts/tests/AUTH/test_3b_key_migration_execute.py

```
@@ -0,0 +1,527 @@
+    # Error case 1: Invalid pubkey format (not 64-char hex)
+    print("\n[TEST] Invalid pubkey format (short hex)...")
+    invalid_request = {
+        "new_admin_pubkey": "abc123",  # Too short
+        "users": [],
+        "field_values": [],
+        "signature_event": create_signed_auth_event(old_admin_privkey, admin_pubkey, "admin_key_migration")
+    }
+    try:
+        response = requests.post(f"{api_base}/admin/key-migration/execute", json=invalid_request, headers=headers, timeout=10)
+        if response.status_code == 400:
+            print(f"  Result: 400 Bad Request (expected)")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)